### PR TITLE
style and copy edits

### PR DESCRIPTION
**Pull request by awfrancisco**

### CODE REVIEW

File: pages/education-and-advocacy.md [hunk 1]
```diff
@@ -89,28 +89,28 @@ Chasm” model and Albert Wong’s framework image._
...
 
-## <a name="poor-quality-not-an-issue"></a>“Poor Quality” Code Not an Issue
+## <a name="poor-quality-not-an-issue"></a>“Poor quality” code not an issue
 
-“Poor quality” code is not part of the chasm. It is a symptom, not a cause.
+“Poor quality” code is not part of the chasm. It's a symptom, not a cause.
 Spread ideas and remove obstacles, and the code will improve.
 
-## <a name="googles-testing-grouplet"></a>Case Study: Google’s Testing Grouplet
+## <a name="googles-testing-grouplet"></a>Case study: Google’s testing grouplet
```
Comment (mbland) [inline pages/education-and-advocacy.md]: "Testing Grouplet" is a proper noun.

File: pages/education-and-advocacy.md [hunk 2]
```diff
@@ -89,28 +89,28 @@ Chasm” model and Albert Wong’s framework image._
...
 
-Google's **Testing Grouplet** was a team of volunteers pooling their 20% time
+Google's **Testing Grouplet** was a team of volunteers pooling their 20 percent time
```
Comment (mbland) [inline pages/education-and-advocacy.md]: I'd leave this "20%".

File: pages/education-and-advocacy.md [hunk 3]
```diff
@@ -89,28 +89,28 @@ Chasm” model and Albert Wong’s framework image._
 
 Fulfilling each step is necessary to producing lasting, positive change. By
 moving from one step to the next (conceptually; in reality many efforts must
-happen in parallel), the Early Majority is transformed from “Dependent” upon
-the Instigators to “Independent”:
+happen in parallel), the Early Majority is transformed from dependent upon
```
Comment (mbland) [inline pages/education-and-advocacy.md]: "Dependent" and "Independent" here were referring to the labels on the Rainbow of Death graphic. Not sure if they should stay as they were, or as you've updated them here.

File: pages/processes-and-artifacts.md [hunk 4]
```diff
@@ -22,25 +22,25 @@ maintain momentum, productivity, and morale.
...
 
-## <a name="two-pager"></a>The Two-Pager
+## <a name="two-pager"></a>The two-pager
 
 When faced with the task of forming a new grouplet, a very useful exercise is
-to write a two-page document describing the [Mission](#mission) of the
-grouplet, its [Objectives and Key Results](#okrs), and the initial set of
-[Roles](#roles) that will form the grouplet's basic structure. This two-pager
+to write a two-page document describing the [mission](#mission) of the
+grouplet, its [objectives and key results](#okrs), and the initial set of
```
Comment (mbland) [inline pages/processes-and-artifacts.md]: "Objectives and Key Results" is a proper noun.

File: pages/working-groups.md [hunk 5]
```diff
@@ -1,39 +1,39 @@
...
 * Identify a mission, objectives, and projects.  
 * Weigh quantitative or qualitative measurements.  
-* Most working groups will have one or two leaders; however, some working groups
+* Most working groups will have one or two leaders. However, some working groups
 may be a way for multiple related teams to have open channels of communication,
 such as #wg-training, which provides a place for various training initiatives
 to report on progress, for individuals to ask for help and interested experts
 to give feedback on training initiatives.
-* Each working group should post a general description of the group's purpose in the slack channel description and set 1 or more initial goals.  Goals can be added or updated at any time and removed with done or it is decided that they are no longer goals.
+* Each working group should post a general description of the group's purpose in the Slack channel description and set ong or more initial goals. Goals can be added or updated at any time and removed when done or it's decided that they're no longer goals.
```
Comment (mbland) [inline pages/working-groups.md]: Typo: `ong` => `one`.

**Comment by mbland** [comment]
Thanks for doing this, @awfrancisco! Just a few nits; feel free to merge after addressing.

**Comment by awfrancisco** [comment]
I accepted all your edits. Will merge now.

**Pull request merged**